scripts/bash/run_enceladus.sh

```diff
@@ -440,6 +447,9 @@ add_to_cmd_line "--experimental-mapping-rule" ${EXPERIMENTAL_MAPPING_RULE}
 add_to_cmd_line "--catalyst-workaround" ${CATALYST_WORKAROUND}
 add_to_cmd_line "--autoclean-std-folder" ${AUTOCLEAN_STD_FOLDER}
 add_to_cmd_line "--persist-storage-level" ${PERSIST_STORAGE_LEVEL}
+if [ "$HELP_CALL" == "1" ]; then
+    CMD_LINE="$CMD_LINE --help"
```

Comment:
It took me a while to understand that `--help` operates on a different level than one would conventionally expect. I believe that the expectation for `--help` argument is that:

1. it will prevent the program from running normally (transitively kind of true)
2. it will give helpful information on the program/command **directly** called (not really true)

Though seeming logically correct, my opinion is that the **argument should be named differently** (e.g. `--pass-help`), while `--help` would be kept reserved if we ever want to offer to provide first-hand help information for the script itself.

Reply:
All the parameters are passed to the Spark job eventually, well except the `-`dry-run`.
Although the `--pass-help` would be something not intuitive, requiring extra documentation which kind of contradicts the parameter's purpose 😉 
Regarding 2, - right now the help in _SparkJobs.jar_ is broken. It's being addressed (actually already fixed) in @AdrianOlosutean PR #1511

Reply:
But `--help` parameter is usually somewhat special as it prevents running and processing other params (there can be exceptions). 

I do not, however, strongly object to this, logically it makes sense. I just felt the need to call out the disparity between the function and possible user's expectation. After I have pointed this out and you still think this is the best form, feel free to merge it as is. Oh snap, you already have 😆